pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,200 @@
+* `BuildkiteAgentQueue` the Buildkite Queue your Pipeline Steps target
+
+There are also optional parameters to configure the size of the Auto Scaling
+Group:
+
+* `MinSize` defaults to 0
+* `MaxSize` defaults to 3
+
+The default AWS Limit for `mac1.metal` is 3 dedicated hosts per account region. If
+you require more than 3 instances, request an increased limit in the *AWS
+Service Quotas Dashboard*.
+
+### Deploy Using the AWS Console
+
+Use the launch button below to Create a CloudFormation stack from the latest
+version of our template:
+
+<a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite-mac&templateURL=https://s3.amazonaws.com/buildkite-serverless-apps-us-east-1/elastic-mac/template/latest.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
+
+Ensure the selected region in the top menu bar matches the region of your VPC
+and AMI resources.
+
+Give your stack a unique name, and fill in the required parameters.
+
+### Deploy Using the AWS CLI
+
+To deploy using the AWS CLI, save your parameters in a `.parameters.json` file
+and run the following commands:
+
+```
+$ cat .parameters.json
+> [
+	{
+		"ParameterKey": "ImageId",
+		"ParameterValue": "ami-0c3a7d0c15048b6b5"
+	},
+	{
+		"ParameterKey": "RootVolumeSize",
+		"ParameterValue": "250"
+	},
+	{
+		"ParameterKey": "Subnets",
+		"ParameterValue": "subnet-f3e72abb,subnet-f23fe294"
+	},
+	{
+		"ParameterKey": "SecurityGroupIds",
+		"ParameterValue": "sg-a09db9d7"
+	},
+	{
+		"ParameterKey": "BuildkiteAgentQueue",
+		"ParameterValue": "mac"
+	},
+	{
+		"ParameterKey": "BuildkiteAgentToken",
+		"ParameterValue": "[redacted]"
+	}
+]
+
+$ make
+> sed "s/%v/v0.0.1-9-g1790b0d/" <template.yml >build/template.yml
+
+$ aws cloudformation deploy --stack-name buildkite-mac --region YOUR_REGION --template-file build/template.yml --parameters-override file:///$PWD/.parameters.json
+```
+
+See the [AWS CloudFormation Deploy CLI documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/deploy/index.html)
+for more details.
+
+## Starting your Buildkite Agents
+
+Once you have successfully deployed the template, use the deployed stack’s
+*Resources* tab to find the `AutoScaleGroup` and open the *Physical ID* link.
+*Edit* the selected Auto Scaling group, and set the *Desired capacity* to the
+number of instances you require.
+
+The Auto Scaling group will automatically provision Dedicated Hosts using the
+Host Resource Group and boot instances on them. The Launch Template’s `UserData`
+script will install, configure, and start the Buildkite Agent.
+
+`mac1.metal` instances are slow to boot and slow to terminate. If want to match
+your *Desired capacity* to your workload, consider using an Auto Scale group
+*Scheduled action*.
+
+## Frequently Asked Questions
+
+### My ASG doesn’t launch any instances
```

Comment:
```suggestion
### My Auto Scaling group doesn’t launch any instances
```

Reply:
auto-scaling?

Reply:
AWS refer to it capital cased so I think we should go with that https://aws.amazon.com/autoscaling/ 😄